src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -2752,45 +2752,6 @@ static void Main(string[] args)
     Property("Length"));
         }
 
-        [WpfTheory(Skip = "https://github.com/dotnet/roslyn/issues/30855")]
-        [CombinatorialData]
-        [WorkItem(18956, "https://github.com/dotnet/roslyn/issues/18956")]
-        public async Task TestVarInPattern1(TestHost testHost)
```

Comment:
Why is this test removed?

Reply:
The classification is no longer semantic for this kind of `var`. It is syntactic. So the coverage comes from syntactic classifier tests now.

Reply:
I'd personally keep the test and document the current behavior. But @CyrusNajmabadi for suggestion.